internal/ld/ld.go

```diff
@@ -80,6 +82,10 @@ func InitApiClient(options ApiOptions) ApiClient {
 	if options.RetryMax != nil && *options.RetryMax >= 0 {
 		client.RetryMax = *options.RetryMax
 	}
+	client.RetryWaitMin = 1000 * time.Millisecond
+	client.RetryWaitMax = 30000 * time.Millisecond
```

Comment:
TIL `time.Duration` is in nanoseconds o__O

...so my brain is malfunctioning 😅 

RetryWaitMin = 1 ms
RetryWaitMax = 30 ms

am i reading that right?

Reply:
my bad i was forgetting 0s. what if we did this to make it more clear?

```suggestion
	client.RetryWaitMin = 1 * time.Second
	client.RetryWaitMax = 30 * time.Second
```